src/otx/v2/adapters/datumaro/adapter/datumaro_dataset_adapter.py

```diff
@@ -212,7 +212,7 @@ def _import_dataset(
         return dataset
 
     @abstractmethod
-    def get_otx_dataset(self) -> DatasetEntity:
+    def get_otx_dataset(self) -> Dict[Subset, DatumDataset]:
```

Comment:
Oh, why do we still need the `Subset` of `DatasetEntity`?

Reply:
Subset concept was defined during import_dataset (this is because, OTX tries to import each subset at the initial). I am planning to remove Subset later by fixing import logics. If so, current DatasetEntity can be purely replaced with DatumDataset. Thank you.